process/app.py

```diff
@@ -178,8 +180,8 @@ def process_message(tts_domain, tts_api_key, post_data):
             properties = layer.properties
             if 'geometryType' in properties and layer.properties['geometryType'] == 'esriGeometryPoint':
                 print("Updating GeometryPoint")
-
-                where_clause = f"name='{name}'"
+                escapedName = name.replace("'", "''")
+                where_clause = f"name='{escapedName}'"
```

Comment:
Nice catch!

Maybe move these two lines to above the if, so that we do not need to maintain it in two places. Note the use of snake_case.

e.g:
```
        for layer in arcgis_item_last.layers:
            escaped_name = name.replace("'", "''")
            where_clause = f"name='{escaped_name}'"
            properties = layer.properties

            if 'geometryType'...
```